sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/datetimeExpressions.scala

```diff
@@ -96,7 +96,7 @@ case class CurrentTimestamp() extends LeafExpression with CodegenFallback {
   override def dataType: DataType = TimestampType
 
   override def eval(input: InternalRow): Any = {
-    System.currentTimeMillis() * 1000L
+    instantToMicros(Instant.now())
```

Comment:
BTW, JDK-8068730 is for JDK9, and it's unclear to me if this is applicable in JDK8 environment. Could you add more clear explanation about JDK8 in the PR description please?

Reply:
`Instant.now()` exists in Java 8 but yeah the behavior change is only Java 9.

Reply:
`Instant.now()` returns a timestamp with millisecond resolutions in JDK8 (at least in 1.8.0.202) by using `System.currentTimeMillis()`.

Reply:
Thank you for updating!